examples/server/server.cpp

```diff
@@ -729,10 +760,16 @@ struct llama_server_context
                     img_sl.img_data = clip_image_u8_init();
                     if (!clip_image_load_from_bytes(image_buffer.data(), image_buffer.size(), img_sl.img_data))
                     {
-                        LOG_TEE("slot %i - failed to load image [id: %i]\n", slot->id, img_sl.id);
+                        LOG_WARNING("failed to load image", {
```

Comment:
In this error case, we `return false;` which I don't know if it will returns a HTTP error message or not.

If that's the case, I think it's better to `LOG_ERROR` here

Reply:
It's a client error, so I would prefer to keep a WARNING, probably the image format is incorrect. Yes task result error is checked in "almost" all http handler.